Magento_Theme/web/js/custom-script.js

```diff
@@ -0,0 +1,18 @@
+let noRouteParagraph = document.querySelector('p');
```

Comment:
`querySelector` returns first found `p` element on the page. are you sure that you have only one on the page?
it'd be better to do something like
```js
const parentRow = document.querySelector('.pagebuilder-custom-row');
const p = parentRow.querySelector('p');
```